### ISSUE TITLE
Is there any plan to support TeaCache for training-free acceleration?

### ISSUE THREAD

**Issue by LiewFeng** [huggingface/diffusers#10374]
TeaCache is a training-free inference acceleration method for visual generation. TeaCache currently supports HunyuanVideo, CogVideoX, Open-Sora, Open-Sora-Plan and Latte. TeaCache can speedup HunyuanVideo 2x  without much visual quality degradation.  For example, the inference for a 720p, 129-frame video takes around 50 minutes on a single A800 GPU while TeaCache can sppeedup to 23 minutes.  Thanks for your efforts!
https://github.com/LiewFeng/TeaCache.

**Comment by a-r-r-o-w** [comment]
Yes, we're definitely working towards integrating this by next diffusers release! Thanks for sharing :hugs:

**Comment by LiewFeng** [comment]
Looking forward to the next release!

**Comment by LiewFeng** [comment]
Hi, @a-r-r-o-w . TeaCache now supports Mochi, LTX-Video, FLUX and LuminaT2X based on diffusers, which may help.
